volto/src/addons/volto-climatechange-elements/src/actions/index.js

```diff
@@ -73,9 +73,31 @@ export function getSiteTitle() {
     type: GET_SITE_TITLE,
     request: {
       op: 'get',
-      path: '@controlpanels/cmsconf-controlpanel',
+      path: '@cmsconf-site_title',
       headers: {},
     },
     url: '',
   };
 }
+export function getGA_ID() {
+  return {
+    type: GET_GA_ID,
+    request: {
+      op: 'get',
+      path: '@cmsconf-ga_id',
+      headers: {},
```

Comment:
I'd be tempted to set `Content-Type: application/json` on these new json endpoints we're making (and the previous ones).